packages/grafana-ui/src/components/ColorPicker/ColorPickerPopover.tsx

```diff
@@ -7,6 +7,7 @@ import { warnAboutColorPickerPropsDeprecation } from './warnAboutColorPickerProp
 import { css } from '@emotion/css';
 import { GrafanaTheme2 } from '@grafana/data';
 import { stylesFactory, withTheme2 } from '../../themes';
+import { formatHexByAlpha } from '../../../../grafana-data/src/themes/colorManipulator';
```

Comment:
import from `@grafana/data` -- I think you will need `colorManipulator.formatHexByAlpha`

Reply:
without our packages (grafana/data, grafana/ui, etc) we need to be careful to import the others with packages -- the build may work, but behaves strangely.

In grafana core (ie, public/app/...) you can import directly, but only do it if the original value should not be exported